element-hq/element-x-android
# Add legacy call invite state events and notifications

## Type of change


## Content

- Add a legacy call invite state event to the timeline.
- Display a notification when a legacy call invite is received.

## Motivation and context

Closes #2485

## Screenshots / GIFs

In the PR (most of them are accidental renames thanks to Showkase...).

## Tests



- From another client, start a legacy (jitsi) call in a DM.
- Check the notifications arrives and is displayed in EXA.
- Open the notification, check the a state event-like timeline item is displayed for it.

## Tested devices

- OS version(s): 14

## Checklist
